src/home/components/PetVetPage/index.js

```diff
@@ -117,6 +121,10 @@ export default function PetVetPage({ vaccines = [] } = {}) {
   this.button.mount($footer);
 
   const emitForm = () => {
+    if (form.isSpecialCare === undefined || form.isNeutered === undefined) {
+      return;
+    }
```

Comment:
Lembre-se que `undefined`, `null`, `false`, `''` e `0` são todos _falsy_ values, então podemos simplificar esse `if`

```suggestion
    if (!form.isSpecialCare || form.isNeutered) return;
```